default.nix

```diff
@@ -75,13 +75,21 @@ stdenv.mkDerivation rec {
         (path: _:
            !elem (baseNameOf path) [".git" "result" "bin" "_build" "_build_ci"]) ./.;
 
+  enableParallelBuilding = true;
```

Comment:
The reason not to put this was because `world` and `byte` are not supposed to be run together in parallel. It would probably be more reasonable to wait to use Dune to build before enabling parallel building.